VSCode/armsnippets.json

```diff
@@ -26,13 +26,6 @@
     ],
     "description": "Skeleton ARM Parameters Template"
 },
-"Variable": {
```

Comment:
Variables are different from parameters - let's keep both

Reply:
Your absolutely right! Me not thinking at all!